### ISSUE TITLE
yhat for ranger is not working properly for classification data

### ISSUE THREAD

**Issue by pbiecek** [ModelOriented/DALEX#163]
Error in `DALEX:::yhat.ranger` 
`Error in if (ncol(pred) == 2) { : argument is of length zero`

full example:

```
library("DALEX")
library("ranger")

titanic_rf <- ranger(survived ~ class + gender + age + sibsp + 
         parch + fare + embarked, data = titanic_imputed,
         classification = TRUE)

titanic_ex <- explain(titanic_rf,
                data  = titanic_imputed,
                y     = titanic_imputed$survived,
                label = "Regression Forest for Titanic")

```

**Comment by pbiecek** [comment]
The model was wrong, it should be

```
library("ranger")
titanic_rf <- ranger(survived ~ class + gender + age + sibsp + 
         parch + fare + embarked, data = titanic_imputed,
         probability = TRUE,
         classification = TRUE)
titanic_rf
```

**Comment by maksymiuks** [comment]
But it will be good idea to ensure that yhats handle  vector input

**Comment by pbiecek** [comment]
fixed in d899fd9
